dotnet/src/SemanticKernel.Test/TemplateEngine/PromptTemplateEngineTests.cs

```diff
@@ -11,23 +12,20 @@
 using Moq;
 using SemanticKernelTests.XunitHelpers;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace SemanticKernelTests.TemplateEngine;
 
-public sealed class TemplateEngineTests : IDisposable
+public sealed class TemplateEngineTests
```

Comment:
nit: can we update class name to `PromptTemplateEngineTests` to match class name which we test and also file name?

Reply:
Sure, updated + some cleanup